server/data/migrations/20190213174049_classes_teachers.js

```diff
@@ -8,7 +8,7 @@ exports.up = function(knex, Promise) {
       .integer('class_id')
       .references('id')
       .inTable('classes');
-    tbl
+    /*tbl
```

Comment:
Maybe delete the comments if we don't need them (keep them if you're still using them, just trying to be a second set of eyes).

Reply:
Yeah, thanks! Will do, just wanted to be sure I wasn't breaking anything.